internal/pkg/service/stream/api/mapper/task.go

```diff
@@ -11,7 +11,15 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-func (m *Mapper) NewTaskResponse(entity task.Task) *api.Task {
+type taskOutputs struct {
+	URL       *string            `mapstructure:"url,omitempty"`
+	ProjectID *keboola.ProjectID `mapstructure:"projectId,omitempty"`
+	BranchID  *keboola.BranchID  `mapstructure:"branchId,omitempty"`
+	SourceID  *key.SourceID      `mapstructure:"sourceId,omitempty"`
+	SinkID    *key.SinkID        `mapstructure:"sinkId,omitempty"`
+}
```

Comment:
I simplified the code bellow, there is no more a map with strings keys, but the `taskOutputs` struct.